### ISSUE TITLE
[JENKINS-67965] No focus state on new checkboxes

### ISSUE THREAD

**Issue by jenkins-infra-bot** [timja-org/jenkins-gh-issues-poc-11-13#912]
<p>Radios have a focus state:</p>

<p> <span class="image-wrap" style=""><a id="57399_thumb" href="https://issues.jenkins.io/secure/attachment/57399/57399_image-2022-03-06-11-16-51-033.png" title="image-2022-03-06-11-16-51-033.png" file-preview-type="image" file-preview-id="57399" file-preview-title="image-2022-03-06-11-16-51-033.png"><img src="https://issues.jenkins.io/secure/thumbnail/57399/_thumb_57399.png" style="border: 0px solid black" role="presentation"/></a></span> </p>

<p>Checkboxes do not:<br/>
 <span class="image-wrap" style=""><a id="57398_thumb" href="https://issues.jenkins.io/secure/attachment/57398/57398_image-2022-03-06-11-17-21-994.png" title="image-2022-03-06-11-17-21-994.png" file-preview-type="image" file-preview-id="57398" file-preview-title="image-2022-03-06-11-17-21-994.png"><img src="https://issues.jenkins.io/secure/thumbnail/57398/_thumb_57398.png" style="border: 0px solid black" role="presentation"/></a></span> </p>

<p>cc <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=janfaracik" class="user-hover" rel="janfaracik">janfaracik</a></p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=timja">timja</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-67965" target="_blank">No focus state on new checkboxes</a></i></summary>
<i><ul>
<li><b>status</b>: Closed
<li><b>priority</b>: Minor
<li><b>resolution</b>: Fixed
<li><b>resolved</b>: 2022-03-08T15:00:46+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 1
<li><b>imported</b>: 2025-11-13
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>Radios have a focus state:</p>

<p> <span class="image-wrap" style=""><a id="57399_thumb" href="https://issues.jenkins.io/secure/attachment/57399/57399_image-2022-03-06-11-16-51-033.png" title="image-2022-03-06-11-16-51-033.png" file-preview-type="image" file-preview-id="57399" file-preview-title="image-2022-03-06-11-16-51-033.png"><img src="https://issues.jenkins.io/secure/thumbnail/57399/_thumb_57399.png" style="border: 0px solid black" role="presentation"/></a></span> </p>

<p>Checkboxes do not:
 <span class="image-wrap" style=""><a id="57398_thumb" href="https://issues.jenkins.io/secure/attachment/57398/57398_image-2022-03-06-11-17-21-994.png" title="image-2022-03-06-11-17-21-994.png" file-preview-type="image" file-preview-id="57398" file-preview-title="image-2022-03-06-11-17-21-994.png"><img src="https://issues.jenkins.io/secure/thumbnail/57398/_thumb_57398.png" style="border: 0px solid black" role="presentation"/></a></span> </p>

<p>cc <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=janfaracik" class="user-hover" rel="janfaracik">janfaracik</a></p></pre>
</details>
</details>
<ul><li><i>environment</i>: <code>2.337</code></li></ul>
<details><summary><i>2 attachments</i></summary>

- [image-2022-03-06-11-16-51-033.png](https://issues.jenkins.io/secure/attachment/57399/image-2022-03-06-11-16-51-033.png)
  > ![image-2022-03-06-11-16-51-033.png](https://issues.jenkins.io/secure/attachment/57399/image-2022-03-06-11-16-51-033.png)
- [image-2022-03-06-11-17-21-994.png](https://issues.jenkins.io/secure/attachment/57398/image-2022-03-06-11-17-21-994.png)
  > ![image-2022-03-06-11-17-21-994.png](https://issues.jenkins.io/secure/attachment/57398/image-2022-03-06-11-17-21-994.png)
</details>
